lms/djangoapps/api_manager/users/views.py

```diff
@@ -17,6 +18,7 @@
 
 from courseware import grades, module_render
 from courseware.model_data import FieldDataCache
+from courseware.views import get_module_for_descriptor, save_child_position, get_current_child
```

Comment:
@mattdrayer I tried to only touch these modules minimally-- I fixed a couple of things on the API side because they were getting rather distracting, but didn't do as much cleanup. I'll go ahead and clear out those unused imports at least.